dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/util/AnnotationUtils.java

```diff
@@ -348,12 +348,12 @@ public static String resolveInterfaceName(Reference reference, Class<?> defaultI
              * @since 2.7.1
              * ignore annotation member
```

Comment:
There should be a place to get an annotation to do the parsing error.

Reply:
This is a  annotation util, i think it should  be universal, so I couldn't understand why this method ignore annotation member

Reply:
@cvictory  i need you help

Reply:
I have confirmed this question with cvictory. What is the reason why he is not too clear about it. It was because there was an explanation error at the time of starting. At the time, there was no such problem.

Reply:
tks.